src/plugins/intel_gpu/src/plugin/transformations/binary_conv_to_conv.cpp

```diff
@@ -0,0 +1,120 @@
+        auto fp_element_type = activations.get_element_type();
+
+        ov::Tensor new_weights_data(fp_element_type, weights->get_output_shape(0));
+        auto src_ptr = static_cast<const uint8_t*>(weights->get_data_ptr());
+        auto size = ov::shape_size(weights->get_shape());
+        switch (fp_element_type) {
+            case ov::element::f16: convert_packed_bin_to_fp(src_ptr, static_cast<ov::float16*>(new_weights_data.data()), size); break;
+            case ov::element::f32: convert_packed_bin_to_fp(src_ptr, static_cast<float*>(new_weights_data.data()), size); break;
+            default: return false;
+        }
+
+        auto new_weights_const = std::make_shared<ov::op::v0::Constant>(new_weights_data);
+        auto rank = activations.get_partial_shape().size();
+
+        auto in_lo = pattern_map.at(in_lo_m);
+        auto in_hi = pattern_map.at(in_hi_m);
+        auto out_lo = std::make_shared<ov::op::v0::Constant>(fp_element_type, ov::Shape(rank, 1), std::vector<float>{-1.0f});
+        auto out_hi = std::make_shared<ov::op::v0::Constant>(fp_element_type, ov::Shape(rank, 1), std::vector<float>{1.0f});
+
+        auto new_fq = std::make_shared<ov::op::v0::FakeQuantize>(activations, in_lo, in_hi, out_lo, out_hi, 2);
+        std::vector<std::shared_ptr<ov::Node>> result_nodes = { new_fq };
+
+        std::shared_ptr<ov::Node> conv_input = new_fq;
+        auto pb = binary_conv->get_pads_begin();
+        auto pe = binary_conv->get_pads_end();
+        if (binary_conv->get_pad_value() != 0.0f) {
+            pb.insert(pb.begin(), rank - pb.size(), 0);
+            pe.insert(pe.begin(), rank - pe.size(), 0);
+            auto pad_b = std::make_shared<ov::op::v0::Constant>(ov::element::i32, ov::Shape{pb.size()}, pb);
+            auto pad_e = std::make_shared<ov::op::v0::Constant>(ov::element::i32, ov::Shape{pe.size()}, pe);
+            auto pad_v = std::make_shared<ov::op::v0::Constant>(fp_element_type, ov::Shape{}, std::vector<float>{binary_conv->get_pad_value()});
+            auto pad = std::make_shared<ov::op::v1::Pad>(new_fq, pad_b, pad_e, pad_v, ov::op::PadMode::CONSTANT);
+            conv_input = pad;
+
+            pb = ov::CoordinateDiff(binary_conv->get_pads_begin().size(), 0);
+            pe = ov::CoordinateDiff(binary_conv->get_pads_end().size(), 0);
+            result_nodes.push_back(pad);
+        }
+        auto convolution = std::make_shared<ov::op::v1::Convolution>(conv_input,
+                                                                     new_weights_const,
+                                                                     binary_conv->get_strides(),
+                                                                     pb,
+                                                                     pe,
+                                                                     binary_conv->get_dilations(),
+                                                                     ov::op::PadType::EXPLICIT);
```

Comment:
Shouldn't we reuse original padding type here?

Reply:
Actually padding is handled by explicit Pad operation above, that's why pad type is changed here. I think generic transformation would require more carful pad processing (e.g. in cases when pad>0 && pav_val == 0.0f), but given that we have only a couple of resnet versions with that operation, I wouldn't want to put more efforts for the generalization of this pass.

Reply:
Okay, thanks for clarification!